packages/main/src/Table.js

```diff
@@ -190,6 +191,17 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
```

Comment:
All the new API should have @ since 1.0.0-rc.13

Reply:
Done